lib/ruby_indexer/test/method_test.rb

```diff
@@ -123,8 +123,9 @@ def bar(a)
 
       assert_entry("bar", Entry::Method, "/fake/path/foo.rb:1-2:2-5")
       entry = T.must(@index["bar"].first)
-      assert_equal(1, entry.parameters.length)
-      parameter = entry.parameters.first
+      parameters = entry.signatures.first.parameters
+      assert_equal(1, parameters.length)
```

Comment:
These kind of length assertions were useful during development, but we remove them to make the tests a little shorter.